frost/coordinator.go

```diff
@@ -64,7 +68,22 @@ func (c *Coordinator) Aggregate(
 	//    - (R, z), a Schnorr signature consisting of an Element R and
 	//      Scalar z.
 
-	// TODO: validate the number of signature shares
+	if len(signatureShares) < c.threshold {
+		return nil, fmt.Errorf(
+			"not enough shares; has [%d] for threshold [%d]",
+			len(signatureShares),
+			c.threshold,
+		)
+	}
```

Comment:
I would love to understand how this relates to the docstring section where we say:
```
- sig_shares, a set of signature shares z_i, Scalar values, for each
  participant, of length NUM_PARTICIPANTS, where
  MIN_PARTICIPANTS <= NUM_PARTICIPANTS <= MAX_PARTICIPANTS.
```
Do we enforce `MIN_PARTICIPANTS`/`MAX_PARTICIPANTS` or just rely on the `threshold`?

Reply:
It is the `MIN_PARTICIPANTS <= NUM_PARTICIPANTS` condition but... why not check the `<= NUM_PARTICIPANTS <= MAX_PARTICIPANTS` condition as well? Added in 375abab6e84a3e031abc5366963fa15155757386.